ArduCopter/avoidance_adsb.h

```diff
@@ -19,6 +19,9 @@ class AP_Avoidance_Copter : public AP_Avoidance {
     // helper function to set modes and always succeed
     void set_mode_else_try_RTL_else_LAND(Mode::Number mode);
 
+    // get the minimum altitude when descend
```

Comment:
`minimum limit altitude allowed on descend ` maybe ?

Reply:
Thank you!